public/src/components/channelManagement/helpers/testEditorStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { createStyles, Theme } from '@material-ui/core';
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const testEditorStyles = ({ spacing, palette }: Theme) =>
+  createStyles({
```

Comment:
Yeah again I think it's great to share styles! But probably best to go for the `useStyles` method. We could rename this file to something like `useTestEditorStyles` and have it export that as a hook